src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx

```diff
@@ -0,0 +1,106 @@
+import { FormPanel } from '@components/interface';
+import { IconLabel, InputTokenLabel, MaskedIntegerField } from '@components/composite';
+import { Box } from '@mui/system';
+import { Panel, Typography } from "src/components/atomic";
+import { AugmentedMellowLpVault, formatCurrency, toUSFormat } from '@utilities';
+import { Button } from '@components/atomic';
+import { colors } from '@theme';
+import { ReactNode } from 'react';
+import { isUndefined } from 'lodash';
+
+export type LPMellowVaultDepositWindowProps = {
+    lpVault: AugmentedMellowLpVault;
+    onChangeDeposit: (value: number | undefined) => void;
+    submitText: string;
+    hintText: ReactNode;
+    onSubmit: () => void;
+    disabled: boolean;
+};
+
+const LPMellowVaultDepositWindow: React.FunctionComponent<LPMellowVaultDepositWindowProps> = ({ lpVault, onChangeDeposit, submitText, hintText, disabled, onSubmit }: LPMellowVaultDepositWindowProps) => {
+
+    const subtext = `WALLET BALANCE: ${isUndefined(lpVault.userWalletBalance) ? "---" : `${formatCurrency(lpVault.userWalletBalance, true)} ${lpVault.tokenName}`}`;
+
+    const handleChange = (newValue: string | undefined) => {
+        const usFormatted = toUSFormat(newValue);
+        onChangeDeposit(!isUndefined(usFormatted) ? parseFloat(usFormatted) : undefined);
+    };
+    
+    const renderContent = () => {
+        return (
+            <FormPanel sx={{ width: '100%', maxWidth: '328px', background: "#19152A", padding: "16px 16px" }}>
+                <Box sx={{ display: 'flex' }}>
+                    <Box sx={{ borderBottom: '1px solid #E5E1F9' }}>
+                        <Typography variant='h1' sx={{ fontSize: '24px', lineHeight: '1.2', color: '#E5E1F9', fontWeight: '700', marginLeft: "0px" }}>
+                            DEPOSIT
+                        </Typography>
+                    </Box>
+
+                    <Box sx={{ borderBottom: '1px solid #2A2444', marginLeft: "16px" }}>
+                        <Typography variant='h1' sx={{ fontSize: '24px', lineHeight: '1.2', color: '#2A2444', fontWeight: '700', marginLeft: "0px" }}>
+                            WITHDRAW
+                        </Typography>
+                    </Box>
+                </Box>
+
+                <Typography variant='body1' sx={{ fontSize: '16px', lineHeight: '1.6', color: '#E5E1F9', marginTop: "8px", fontWeight: '400' }}>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Volutpat ut proin porttitor sit lorem.
```

Comment:
please update with proper copy

Reply:
Let's add the copies during the QA process so Simon/Artur/Gerard can see where this is placed.